### ISSUE TITLE
Feature Request - Sangrand & Gurpurab's Hukumnama

### ISSUE THREAD

**Issue by davchana** [Sarabveer/gurbaninow#25]
Hi, is it possible to include this request in a future version of API?

To request Hukumnama but instead of daily one, of Sangrad Day; as it gets issued from Harmandir Sahib Ji.

Points to consider:

1.  How to ask? Each Sangrand is 1st of desi month; total 12 months in desi Calender ([List from Wikipedia](https://en.wikipedia.org/wiki/Bikrami_calendar#Solar_calendar)). if required for Vaisakh (1st Month), aqpi endpoint can be `/sangrand/1` or `sangrand/01` & so on. 
1.  SikhNet, the existing provider of daily Hukamnama to GurbaniNow API does not provide sangrand's one (Does it?). But it is available at Sikhnet's source `http://sgpc.net/sangrand-mukhwak/`, pdf format only.

**Comment by sarabveer** [comment]
I could make the API fetch Sangrand Hukamnamas based on the Mool Nanakshahi Calendar.

SikhNet does not provide Sangrand Hukamnamas AFAIK

**Comment by davchana** [comment]
> I could make the API fetch Sangrand Hukamnamas based on the Mool Nanakshahi Calendar.

@Sarabveer yes, please do that, that would be so nice of you :) Please open an issue to discuss the api endpoint and requests.

Yes, Sikhnet does not, but their source SGPC provides it at above link, but only in pdf format.

I guess the source does not matter as long the text if from Shri Harmandir Sahib Ji. Sikhnet or SGPC or any other.

Thanks.

**Comment by sarabveer** [comment]
I can fetch archives from Sikhnet, so its possible.

**Comment by sarabveer** [comment]
It seems like the Sangrand Hukamnama is just the relative Pauri from Barah Mah Majh for the Desi Month.

**Comment by sarabveer** [comment]
Depends on: https://github.com/ShabadOS/database/issues/1237
